File: Leetcode/cpp/997.cpp
class Solution {
public:
  int findJudge(int n, vector<vector<int>>& trust) {
    vector<int> trusted(n, 0), trusts(n, 0);
    for (int i = 0; i < trust.size(); i++) {
      int from, to;
      from = trust[i][0];
      to = trust[i][1];
      trusted[to - 1]++;
      trusts[from - 1]++;
    }
    for (int i = 0; i < n; i++) {
      if (trusts[i] == 0 && trusted[i] == n - 1) {
        return i + 1;
      }
    }
    return -1;
}
};

File: Leetcode/Python/_997.py
class Solution:
    def findJudge(self, n: int, trust: List[List[int]]) -> int:
        trust_to = [0] * n
        trust_from = [0] * n
        for from_, to_ in trust:
            trust_to[to_ - 1] += 1
            trust_from[from_ - 1] += 1
            
        for i in range(n):
            if trust_to[i] == n - 1 and trust_from[i] == 0:
                return i + 1
        return -1
